critiquebrainz/data/dump_manager.py

```diff
@@ -241,83 +270,74 @@ def public(location, rotate=False):
         tar.add(os.path.join(temp_dir, 'TIMESTAMP'), arcname='TIMESTAMP')
         tar.add(os.path.join(temp_dir, 'SCHEMA_SEQUENCE'), arcname='SCHEMA_SEQUENCE')
 
-        print(" + %s/cbdump.tar.bz2" % dump_dir)
+        return " + %s/cbdump.tar.bz2" % dump_dir
 
-    # REVIEWS
-    # Archiving review tables (review, revision)
 
-    # 1. COMBINED
-    # Archiving all reviews (any license)
-    REVISION_COMBINED_SQL = """
-        SELECT {columns} FROM revision JOIN review
+def create_reviews_archive(dump_dir, temp_dir, license_id=None):
+    """Creates a dump of reviews filtered on the given license_id, their revisions and
```

Comment:
This docstring should be improved as well. You should also add what happens when `license_id=None`.